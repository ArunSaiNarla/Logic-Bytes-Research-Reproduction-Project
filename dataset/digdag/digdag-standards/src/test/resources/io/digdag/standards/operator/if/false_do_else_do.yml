_command: false
_do:
  echo>: do called
_else_do:
  echo>: else_do called

