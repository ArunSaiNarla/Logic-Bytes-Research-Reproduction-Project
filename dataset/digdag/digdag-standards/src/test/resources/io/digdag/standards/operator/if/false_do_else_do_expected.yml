echo>: else_do called
